<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>602</width>
        <height>568</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>602</width>
        <height>568</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionWalls"/>
    <addaction name="actionNo_walls"/>
    <addaction name="separator"/>
    <addaction name="actionChange_snake_color"/>
    <addaction name="actionAdd_image_on_head"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWalls">
   <property name="text">
    <string>Walls</string>
   </property>
  </action>
  <action name="actionNo_walls">
   <property name="text">
    <string>No walls</string>
   </property>
  </action>
  <action name="actionChange_snake_color">
   <property name="text">
    <string>Change snake color</string>
   </property>
  </action>
  <action name="actionAdd_image_on_head">
   <property name="text">
    <string>Add image on head</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
